pkg/libb2/patches/0001-rename-config.h-to-libb2_config.h.patch

```diff
@@ -0,0 +1,26 @@
+From 213f6bb69962154a890925124d44aa8f3361270e Mon Sep 17 00:00:00 2001
```

Comment:
Please modify the description of PR to reflect what it really does. Maybe, the title should not refer ESP8266 since the problem solved with the patch seems to be more general.

Reply:
Ok.